src/nlp.jl

```diff
@@ -593,7 +593,7 @@ function solvenlp(m::Model, traits; suppress_warnings=false)
 end
 
 # getValue for nonlinear subexpressions
-function getValue(x::Union(ReverseDiffSparse.ParametricExpressionWithParams,ReverseDiffSparse.ParametricExpression{0}))
+@compat function getValue(x::Union{ReverseDiffSparse.ParametricExpressionWithParams,ReverseDiffSparse.ParametricExpression{0}})
```

Comment:
What's the effect of applying ``@compat`` to the whole function? Would it be better to do:
```
function getValue(x::@compat(Union{ReverseDiffSparse.ParametricExpressionWithParams,ReverseDiffSparse.ParametricExpression{0}}))
```